src/components/text-input/TextInput.js

```diff
@@ -100,11 +102,19 @@ class TextInput extends Component {
 
     computeLevel = () => {
         const { lineStrength } = this.props;
-        const normalizedStrengthValue = Math.round(LEVELS_NAME.length * lineStrength);
+        const normalizedStrengthValue = Math.ceil(LEVELS_NAME.length * lineStrength);
 
         return normalizedStrengthValue > 0 ? LEVELS_NAME[normalizedStrengthValue - 1] : LEVELS_NAME[0];
     };
 
+    handleKeyDown = (event) => {
```

Comment:
You should call `props.onKeyDown` if passed, otherwise attaching a regular `onKeyDown` won't be called.

Anyway, I don't think this addition make sense. People are already used to onKeyDown, onKeyPress, onKeyUp... why add a "narrower" `onEnter` when I can just add `onKeyDown/onKeyUp` with `event.key === 'Enter'` check? If we keep doing things like this, we end up with `onEsc`, `onXXX`, etc.

Reply:
Fair enough, I agree with this point. I'll change it to onKeyDown.

I initially implemented it like this because I only needed a handler for the enter key down event and it would simplify the callback that would need to be passed as a prop.